api/models/comment/notifications.js

```diff
@@ -65,6 +66,7 @@ export const sendDigests = async () => {
       // read time.
       let lastReadAt = user.pivot.get('last_read_at')
       if (lastReadAt) lastReadAt = new Date(lastReadAt)
+      const locale = mapLocaleToSendWithUS(user.get('settings').locale || 'en-US')
```

Comment:
why en-US here but just en elsewhere?

Reply:
sendwithus need FULL locale, so we need to convert before sending